autode/hessians.py

```diff
@@ -289,10 +304,12 @@ def _eigenvalues_to_freqs(lambdas) -> List[Frequency]:
 
         nus = (np.sqrt(np.complex_(lambdas))
                / (2.0 * np.pi * Constants.ang_to_m * Constants.c_in_cm))
+        nus *= self._freq_scale_factor
```

Comment:
nus? Unclear what that is to me

Reply:
Lol does this mean νs?

Reply:
haha – a set of [nu](https://simple.wikipedia.org/wiki/Nu_(letter)#:~:text=Nu%20(uppercase%2Flowercase%20%CE%9D%20%CE%BD,has%20a%20value%20of%2050.) values